.github/workflows/playwright.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     timeout-minutes: 60
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I'll assume this is because you are still testing some stuff out regarding the tests that are failing?